app/views/layouts/_user_header.html.erb

```diff
@@ -0,0 +1,10 @@
+ <%= link_to  user_path(user), class: "d-flex m-4 justify-content-between align-items-center" do %>
+         
+        <div class="profil">
+           <i class="fas fa-user-circle fa-3x"></i>
+        </div>
+        <div class="user border d-flex align-items-center justify-content-between px-4 bordered">
+            <h2><%= user.name%></h2> 
+            <div class="p-0 m-0 align-self-end">Number of poste: 20</div>
+        </div>
```

Comment:
- [ ] Posts number shouldn't be added like this, currently, when I open an application and I add a user in `rails c` when a user is displayed I see 20 posts which he/she didn't add and it doesn't make sense. User should create posts to be able to see them in their pages